rustls/src/crypto/aws_lc_rs/pq/mod.rs

```diff
@@ -62,3 +62,44 @@ const INVALID_KEY_SHARE: Error = Error::PeerMisbehaved(PeerMisbehaved::InvalidKe
 const X25519_LEN: usize = 32;
 const MLKEM768_CIPHERTEXT_LEN: usize = 1088;
 const MLKEM768_ENCAP_LEN: usize = 1184;
+
+#[cfg(bench)]
```

Comment:
Why inline these here instead of keeping them in a `benches/` module?

Reply:
This is aligned with the other benchmarks that live in `src/crypto/ring/kx.rs` in an inline `mod benchmarks`. There's no reason they actually have to be written in "unit benchmark" style, since everything they test are public APIs. But putting them there means they get duplicated easily for both providers.

Reply:
Fair enough.